runtime/src/main/java/io/quarkiverse/semantickernel/semanticfunction/SemanticFunction.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkiverse.semantickernel.semanticfunction;
```

Comment:
Could you rename the package `semanticfunction` to `semanticfunctions` (with an `s`) so it follows the same convention of Semantic Kernel itself https://github.com/microsoft/semantic-kernel/tree/main/java/semantickernel-api/src/main/java/com/microsoft/semantickernel/semanticfunctions